### PR TITLE
Issue 78 create tasks view

### PR DESCRIPTION
**Pull request by K1rk3y**
## Change Summary
Work in progress, will update asap.

### Change Form

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

### CODE REVIEW

**Comment by yunho7687** [comment]
@0XFF-96 could you please review this PR?

File: server/app/serializers.py
Comment (CodeWithTheDoctor) [inline server/app/serializers.py:1]: I noticed there are two serialiser files: seriali*s*er.py and seriali*z*ers.py
Is one of them redundant?

**Review by CodeWithTheDoctor — CHANGES_REQUESTED** [review]
Great work! Was confused by the two serialisers files though, have a look.

File: server/app/serializers.py
Comment (K1rk3y) [reply server/app/serializers.py:1]: I have removed the redundant file.

**Review by yunho7687 — APPROVED** [review]

**Review by CodeWithTheDoctor — APPROVED** [review]

**Review by YunhuiZ — APPROVED** [review]

**Pull request merged**